### PR TITLE
Bureau du commandant - CO office

### PR DESCRIPTION
**Pull request by LC4492**
# About the pull request

An update to the  CO office

# Explain why it's good for the game

If people like, its good, if people don't, it is not

# Testing Photographs and Procedure

image

</details>

# Changelog
:cl:
maptweak: Changes to the CO office: The bathroom now faces into the CO's bedroom, and not to his main office. Victory cabinet have been moved to the Officer's mess because of logical issues. Extra-detail to the office, including an exclusive stamp, table flags and others. The safe is now inside the CO's bedroom, and not in his office. The energy APC is now inside the CO's bedroom, and not in his office. Jones finally have a BED again, or something like that. Other minor changes to objects, such the addition of a cane, a box of glasses for serving guests, etc.
/:cl:

### CODE REVIEW

File: maps/map_files/USS_Almayer/USS_Almayer.dmm [hunk 1]
```diff
@@ -51359,6 +51384,20 @@
 	icon_state = "sterile_green"
 	},
 /area/almayer/medical/containment)
+"mog" = (
+/obj/structure/safe,
```
Comment (Huffie56) [inline maps/map_files/USS_Almayer/USS_Almayer.dmm:51388]: why not use the safe subtype that i made for CO?
/obj/structure/safe/co_office
it contain everything you are putting in...
it's probably been merge before you started your PR....
main purpose is to be able to change content without having to change map...

Comment (LC4492) [reply maps/map_files/USS_Almayer/USS_Almayer.dmm:51388]: What does it have?

Comment (Huffie56) [reply maps/map_files/USS_Almayer/USS_Almayer.dmm:51388]: image

Comment (Huffie56) [reply maps/map_files/USS_Almayer/USS_Almayer.dmm:51388]: https://github.com/cmss13-devs/cmss13/pull/4574/files

Comment (Huffie56) [reply maps/map_files/USS_Almayer/USS_Almayer.dmm:51388]: so everything you put in it in your change and if you want to add more you can modify the subtype directly...
even made a subtype for the coin...

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Review by Nanu308 — CHANGES_REQUESTED** [review]
bilde

You've made the CO's place look really boxy, we try and avoid blocky squares when possible as it looks plain bad.
Remove the CO's uniform lockdown button, there is one to the south in the hallway right outside his office.
There is a paperbin on the CO's toilet.
There is very little space in front of the CO's office chair now with only 3x tiles. It will be crowded when event's or meetings and such are done there. Shift it back to how it was and get rid of the two file cabinets there.
In the CO's bedroom there are two vendors to the north, same as with south ones place metal flooring/warning stripe usage.
The two flags on the table looks cool, nice edit.
Moving the bathroom to face the bedroom is a good change.
And finally remember, more detailing isn't always better, moderation to hinder things from being full of too much clutter!

bilde
There is a floating table in space. 

Fix the above and I'll go over it again, overall good.

**Comment by LC4492** [comment]
> bilde
> 
> You've made the CO's place look really boxy, we try and avoid blocky squares when possible as it looks plain bad. Remove the CO's uniform lockdown button, there is one to the south in the hallway right outside his office. There is a paperbin on the CO's toilet. There is very little space in front of the CO's office chair now with only 3x tiles. It will be crowded when event's or meetings and such are done there. Shift it back to how it was and get rid of the two file cabinets there. In the CO's bedroom there are two vendors to the north, same as with south ones place metal flooring/warning stripe usage. The two flags on the table looks cool, nice edit. Moving the bathroom to face the bedroom is a good change. And finally remember, more detailing isn't always better, moderation to hinder things from being full of too much clutter!
> 
> bilde There is a floating table in space.
> 
> Fix the above and I'll go over it again, overall good.

The paperbin on the toilet is supposed to be the toilet paper, and it would be in the rack on the side of the toilet. The rest I understand

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Review by Nanu308 — CHANGES_REQUESTED** [review]
Delete the paper bin, we would need a specific sprite to ever add toilet paper..
Remember to unboxify the CO's main office as i said in the original review, add in the two corner walls again.

Shift/delete a plant maybe and move a 2x1 table down there or even put the stuff up north down there.
Good work on the CO's bedroom, but add a couple vents! and it's perfect.

**Comment by LC4492** [comment]
> Delete the paper bin, we would need a specific sprite to ever add toilet paper.. Remember to unboxify the CO's main office as i said in the original review, add in the two corner walls again.
> 
> Shift/delete a plant maybe and move a 2x1 table down there or even put the stuff up north down there. Good work on the CO's bedroom, but add a couple vents! and it's perfect.

Simply doesn't make sense for the things up north to be down in the entrance, its just going to make everything weird!

**Comment by LC4492** [comment]
Done and done

**Review by Nanu308 — APPROVED** [review]
lgtm

**Pull request merged**